.travis.yml

```diff
@@ -19,7 +19,7 @@ install:
 jobs:
   include:
     - stage: build
-      script: 
-        - ./gradlew build
+      script:
+        - ./gradlew build test integrationTest
```

Comment:
Before this PR we were just running `./gradlew build` and the `test` task was still being kicked off. I think this is a result of functionality of the `java` plugin: https://docs.gradle.org/current/userguide/java_plugin.html in that it extends the `build` task to depend on the `check` task, which in turn depends on the `test` task. 

With the explicit call to `test` here Gradle seems to be smart enough to only run the tests once, but this does seem to have some redundancy we should probably resolve, or at least have a better understanding of what's happening here.

We have a section in `build.gradle` for defining task orders (depends), and we seem to be using that for things that we want to run every time we build the app. My opinion is that we continue to use that section to define gradle task orders and depends and that we only call one gradle task from travis. I'm very open to not having this be the case any longer if there is reason to drop this convention.

Thoughts?